modules/builds-running-entitled-builds-with-sharedsecret-objects.adoc

```diff
@@ -6,12 +6,13 @@ You can use a `SharedSecret` object to securely access the entitlement keys of a
 
 The `SharedSecret` object allows you to share and synchronize secrets across namespaces.
 
-:FeatureName: Shared Resource CSI Driver
-include::snippets/technology-preview.adoc[]
+[IMPORTANT]
+====
+The Shared Resource CSI Driver feature is now generally available in link:https://docs.redhat.com/en/documentation/builds_for_red_hat_openshift/1.1[builds for Red Hat OpenShift 1.1]. This feature is now deprecated in the OpenShift Container Platform. To use this feature, ensure you are using builds for Red Hat OpenShift 1.1 or a more recent version.
+====
 
 .Prerequisites
 
-* You have enabled the `TechPreviewNoUpgrade` feature set by using the feature gates. For more information, see xref:../../nodes/clusters/nodes-cluster-enabling-features.adoc#nodes-cluster-enabling[Enabling features using feature gates].
```

Comment:
Is the intent that customers will be following the doc here as if they installed via Builds for OpenShift?